2.4/test/run

```diff
@@ -342,10 +342,121 @@ noprealloc = true" > $config_file
     echo "  Success!"
 }
 
+function insert_and_wait_for_replication() {
+    local host
+    host=$1
+    local data
+    data=$2
+
+    # Storing document into replset and wait replication to finish
+    CONTAINER_IP="${host}" ADMIN_PASS="adminPassword" \
+    mongo_admin_cmd "db.getSiblingDB('test_db').data.insert(${data});
+        while(true) {
```

Comment:
Good idea. I haven't found any easy `sleep` equivalent in JS, so using only counter (it should be enough).